gotests_test.go

```diff
@@ -66,10 +66,13 @@ func TestGenerateTests(t *testing.T) {
 		}, {
 			name: "Target test file",
 			args: args{
-				srcPath: `testdata/test100_test.go`,
+				srcPath:  `testdata/test100_test.go`,
+				only:     regexp.MustCompile("wrapToString"),
+				subtests: true,
 			},
-			wantNoTests: true,
-			wantErr:     true,
+			wantNoTests: false,
+			wantErr:     false,
+			want:        mustReadFile(t, `testdata/test100result/result.go`),
```

Comment:
The target file should live under `testdata/goldens` with a name like `testdata/goldens/target_test_file.go`, as opposed to `testdata/test100result/result.go`.